mix.exs

```diff
@@ -81,21 +81,21 @@ defmodule Xebow.MixProject do
       {:chameleon, "~> 2.2"},
       {:afk, "~> 0.3"},
       {:dialyxir, "~> 1.0.0", only: :dev, runtime: false},
-      {:ex_doc, "~> 0.22.1", only: :dev, runtime: false},
+      {:ex_doc, "~> 0.22", only: :dev, runtime: false},
       {:mox, "~> 0.5", only: :test},
       {:credo, "~> 1.4", only: [:dev, :test], runtime: false, targets: [:host]},
 
       # phoenix + live-view:
       {:floki, ">= 0.0.0", only: :test},
       {:jason, "~> 1.0"},
-      {:phoenix_html, "~> 2.11"},
-      {:phoenix_live_dashboard, "~> 0.2.0"},
+      {:phoenix_html, "~> 2.14"},
+      {:phoenix_live_dashboard, "~> 0.2.7"},
       {:phoenix_live_reload, "~> 1.2", only: :dev, targets: :host},
-      {:phoenix_live_view, "~> 0.13.0"},
+      {:phoenix_live_view, "~> 0.14.1"},
       {:phoenix, "~> 1.5.3"},
```

Comment:
I generally like to keep just major and minor version specs in mix files (leave out the patch), and just rely on the mix.lock to lock in specific patch versions.

Reply:
:+1: Updated